appcelerator-archive/amp
# Using docker secret for JWT secret key

Fixes #1007 
Fixes #1010 
Resolves #1051

## How to test
```
$ docker secret rm amplifier_yml
$ amp cluster create -t local
$ docker secret ls
[Make sure you have an amplifier_yml secret]
```

Integration tests have been updated accordingly.
